ci/codestyle.sh

```diff
@@ -9,4 +9,10 @@ if test -s tabdamage.txt; then
     exit 1
 fi
 
+cd rust
+if ! cargo fmt -- --check; then
+    echo "cargo fmt failed; run: cd rust && cargo fmt" 1>&2
+    exit 1
+fi
```

Comment:
```suggestion
for cargo in $(find rust/ -iname Cargo.toml); do
    if ! cargo fmt --manifest-path $cargo -- --check; then
        echo "cargo fmt failed; run: cd $(dirname $cargo) && cargo fmt" 1>&2
        exit 1
    fi
done
```

Reply:
Thanks, I cargo culted this in, but tweaked slightly.